src/lib/Icons.js

```diff
@@ -5,22 +5,4 @@ const X = () => (
   </div>
 );
 
-const ModalIcon = () => (
-  <svg
-    xmlns="http://www.w3.org/2000/svg"
-    viewBox="0 0 24 24"
-    fill="none"
-    stroke="currentColor"
-    strokeWidth="2"
-    strokeLinecap="round"
-    strokeLinejoin="round"
-    className="dv-icon"
-  >
-    <polyline points="15 3 21 3 21 9" />
-    <polyline points="9 21 3 21 3 15" />
-    <line x1="21" y1="3" x2="14" y2="10" />
-    <line x1="3" y1="21" x2="10" y2="14" />
-  </svg>
-);
-
-export { ModalIcon, X };
+export { X };
```

Comment:
Is this the right way to say this with only one thing? Are the brackets optional in this case?

Reply:
Also a helpful link!